### ISSUE TITLE
Shift+Left-click function of yeta wrench appears to be borked, except on insulated redstone conduits?

### ISSUE THREAD

**Issue by Trisscar05** [SleepyTrousers/EnderIO#3520]
Thread of entire conversation and testing here;
http://forum.feed-the-beast.com/threads/1-5-0-1-10-0-yeta-wrench-shift-left-click-not-working.161372/#post-1701241

It's happened in two different modpacks now, do I figured I'd throw it here in case anyone had seen it before and knew the fix already.

**Comment by HenryLoenwind** [comment]
redstone conduits are the only conduit type that can be forced to connect to anything. all other conduit types can only connect to blocks that can handle items/fluids/computer data/RF.

Is that the problem?

**Comment by Trisscar05** [comment]
I don't think so. It sounds like the issue is that a menu is supposed to appear if you sh/l-c on the conduit that allows you to modify all the sides at once, whether they're connected or not. But it refuses to appear except in the case of the redstone conduits.

**Comment by AnrDaemon** [comment]
No, Crouch-Click is to forcibly open the connector menu. Period. No "all sides" or anything.
It is intended for cases, when the connector itself is not accessible for clicks, that's all.

**Comment by Trisscar05** [comment]
Huh. May I ask what the cardinal direction menu in this video at around 6:26 is then? That would seem to be the missing screen the original poster was referring to.
https://youtu.be/qsaiCirhHUc

**Comment by HenryLoenwind** [comment]
Again, you cannot connect an RF conduit to a workbench or a fluid conduit to a torch. The side selection menu will only be shown if there are 2 or more valid connections possible. If there is exactly one valid connection, then that connector's GUI will be shown directly. A redstone conduit can be attached to anything (including air), so if there's one in the bundle, the side selection menu will be shown always.

**Comment by Trisscar05** [comment]
Mk. I'mma attempt to get the guy who posted the issue to come in and describe things. Probably be easier than me being a go between for two different sites.

**Comment by pdallair** [comment]
Hello, I'm BuzyD in the FTB thread.  Thanks for taking the time to explain things to me.  I'm a little noob when it comes to mods, I didn't know this interface was only accessible under certain conditions and I was easy to jump to conclusion that this wasn't my fault.  

I wanted to connect an IC2 machine to my RF power supply.  I googled a little bit and read that Direwolf20 pack was using EnetBridge so it should work. I thought that I should be able to "force" the connection like I saw direwolf20 do with redstone conduits, at least. It turns out the current version of the pack doesn't even have EnetBridge (DERP! >_<).  I can use rednet energy cables (MFR), it seems, to bridge different power types together.  

Sorry about that, I should have gone into more details on the scenario.  Thanks for your help everyone.
